docs/components/vectordbs/dbs/azure_ai_search.mdx

```diff
@@ -0,0 +1,38 @@
+[Azure AI Search](https://learn.microsoft.com/en-us/azure/search/search-what-is-azure-search/) (formerly known as "Azure Cognitive Search") provides secure information retrieval at scale over user-owned content in traditional and generative AI search applications.
+
+### Usage
+
+```python
+import os
+from mem0 import Memory
+
+os.environ["OPENAI_API_KEY"] = "sk-xx"
```

Comment:
Here can you please also write a comment saying this key is used for embedding purpose.